conf/broker.conf

```diff
@@ -270,6 +270,10 @@ brokerMaxConnections=0
 # The maximum number of connections per IP. If it exceeds, new connections are rejected.
 brokerMaxConnectionsPerIp=0
 
+# Allow schema to be auto updated. If this is disabled, 'is_allow_auto_update_schema' in namespace
```

Comment:
Do we need to add docs to https://pulsar.apache.org/docs/en/next/reference-configuration/#broker and https://pulsar.apache.org/docs/en/next/reference-configuration/#standalone?

Reply:
Added. PTAL.

Reply:
@Jason918 I think the description of the configuration is not correct?